public/script.js

```diff
@@ -0,0 +1,55 @@
+//access elements from html
+const inputSearch = document.querySelector('.input-search')
+const buttSearch = document.querySelector('.btn-search')
+buttSearch.addEventListener('click', () => {
+    const data = inputSearch.value;
+    const options = {
+        method: 'POST',
+        body: JSON.stringify({ name: data }),
+        headers: { 'Content-Type': 'application/json' }
+    }
+    fetch(`/search`, options)
+    .then(res => res.json())
+    .then(data => createCards(data))
+    inputSearch.value=""
+})
+
+let movieSection = document.querySelector(".movies");
+
+  //function to create element
+function createEle(ele, className, parent) {
```

Comment:
the idea behind this function is nice and here is a little enhancement:
```js
function createElement(type, className, child) {
    const element = document.createElement(type);
    element.classList.add(className);
    element.appendChild(child);
    return element;
  }
  
  const title = createElement('p', 'paragraph', 'this is a title')
  
  // I can use this title element in many places
  parent1.appendChild(title)
  parent2.appendChild(title)
```
this way, we have better naming and no need for the comment. Besides if we pass it a child rather than the parent then this gives us the power of using this element as a child for multiple parents(use it in many places, in your case you can use it only in one place and returning the element doesn't make sense since you have already used it by appending it to a specific parent) 

**More enhancements?**
```js
const createElement = (type, options, children) => {
const element = document.createElement(type);
// inject options 
// append children

return element
} 

const title = createElement('p', { class: 'title' }, 'title text content')
const image = createElement('img', { class: 'image', src:'image src' }, null)
const card = createElement('div', { class: 'container' }, [title, image])
```